packages/edit-post/src/components/sidebar/post-initial-panel/index.js

```diff
@@ -0,0 +1,53 @@
+/**
+ * WordPress dependencies
+ */
+import {
+	PanelBody,
+	__experimentalText as Text,
+	__experimentalVStack as VStack,
+} from '@wordpress/components';
+import {
+	store as editorStore,
+	privateApis as editorPrivateApis,
+} from '@wordpress/editor';
+import { decodeEntities } from '@wordpress/html-entities';
+import { useSelect } from '@wordpress/data';
+import { page as pageIcon } from '@wordpress/icons';
+import { __, sprintf } from '@wordpress/i18n';
+import { humanTimeDiff } from '@wordpress/date';
+
+/**
+ * Internal dependencies
+ */
+import { unlock } from '../../../lock-unlock';
+
+const { PostSidebarCard } = unlock( editorPrivateApis );
+
+export default function PostInitialSidebar() {
+	const { modified, title } = useSelect( ( select ) => {
+		const { getEditedPostAttribute } = select( editorStore );
+		return {
+			title: getEditedPostAttribute( 'title' ),
+			modified: getEditedPostAttribute( 'modified' ),
+		};
+	} );
+	return (
+		<PanelBody>
```

Comment:
Should we bundle the panel body within `PostSidebarCard`.
Also maybe we should rename it to `PostCardPanel` or something like that because it would match what we do for `PostFeaturedImagePanel` and similar panels that already exist in the editor package.

Reply:
For me a panel is also related to the ones that have a toggle header and could also be affected by preferences, which is not this case I think.

Additionally a `PanelBody` without a title is adding just a wrapper with some styles, which we could alternatively not use at all - no preference here though.

To sum up, I think we don't need this extra wrapper component and add the code directly where we use it in settings sidebar.

Reply:
I followed @youknowriad recommendation to put PanelBody inside the component because on every use of the component we have a PanelBody this way we remove the code needed for the component.